pkgs/racket-doc/json/json.scrbl

```diff
@@ -80,35 +82,57 @@ the @rfc for more information about JSON.
   the range of @tt{U+10000} and above are encoded as two @tt{\uHHHH}
   escapes, see Section 2.5 of the @|rfc|.
 
+  If @racket[format?] is given as @racket[#t], then @racket[indent] is
```

Comment:
I think you mean to say that if `format?` is true, since you're not checking for just `#t`.